test/DynamoCoreWpfTests/RecordedTests.cs

```diff
@@ -1425,6 +1425,25 @@ public void TestPreviewToggleConnectionMultiOutputNode()
             });
         }
 
+        [Test, RequiresSTA]
+        public void TestAddingAndRemovingConnectors()
```

Comment:
What does this test test? I thought that testing this fix was not possible in the single threaded case?

Reply:
This is testing changing the connectors multiple times. The test was not failing before the fix but would be better to have one more test. So kept this.